components/ingest-service/pipeline/processor/action_project_tagger.go

```diff
@@ -115,7 +115,7 @@ func actionMatchesAllConditions(action *chef.Action, conditions []*authz.Conditi
 	for _, condition := range conditions {
 		switch condition.Attribute {
 		case authz.ProjectRuleConditionAttributes_CHEF_SERVER:
-			if !stringutils.SliceContains(condition.Values, action.RemoteHostname) {
+			if !stringutils.SliceContains(condition.Values, action.ServiceHostname) {
```

Comment:
This and the below file fixes a bug that was "project tagging" actions on the wrong field.